dnsapi/dns_1984hosting.sh

```diff
@@ -150,7 +150,7 @@ _1984hosting_login() {
   _debug2 response "$response"
 
   if _contains "$response" '"loggedin": true'; then
-    One984HOSTING_COOKIE="$(grep -i '^set-cookie:' "$HTTP_HEADER" | _tail_n 1 | _egrep_o 'sessionid=[^;]*;' | tr -d ';')"
+    One984HOSTING_COOKIE="$(grep -io 'sessionid=[^;]*;' "$HTTP_HEADER"  | tr -d ';')"
```

Comment:
don't use `grep -io` options.

please just grep "sessionid=" first, and then grep "set-cookie"

Reply:
Is `grep -i` okay? The last time I did any fixes for the `set-cookie` business I used `grep -i`. If there are any DNS API changes that do not permit `grep -i` I can work with that. I checked the [guide](https://github.com/acmesh-official/acme.sh/wiki/DNS-API-Dev-Guide#12-cross-platform-compatibility-guide), and just `grep -o` is listed as a cross-compatibility issue.